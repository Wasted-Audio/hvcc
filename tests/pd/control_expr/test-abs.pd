#N canvas 827 239 734 565 12;
#X obj 40 22 loadbang;
#X obj 40 190 print;
#X obj 40 80 t b b b;
#X msg 101 115 -3.1;
#X msg 163 115 5.7;
#X msg 40 115 2.3;
#X obj 40 49 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X obj 40 157 expr abs($f1);
#X connect 0 0 6 0;
#X connect 2 0 5 0;
#X connect 2 1 3 0;
#X connect 2 2 4 0;
#X connect 3 0 7 0;
#X connect 4 0 7 0;
#X connect 5 0 7 0;
#X connect 6 0 2 0;
#X connect 7 0 1 0;
