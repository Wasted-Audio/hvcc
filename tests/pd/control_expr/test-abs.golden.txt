[@ 0.000] print: 5.7
[@ 0.000] print: 3.1
[@ 0.000] print: 2.3
